bitcoin/tx.c

```diff
@@ -326,7 +329,7 @@ static void push_tx(const struct bitcoin_tx *tx,
 	assert(res == WALLY_OK);
 	serialized = tal_arr(tmpctx, u8, len);
 
-	res = wally_tx_to_bytes(tx->wtx, flag, serialized, len, &written);
+	res = wally_tx_to_bytes(tx->wtx, flag & WALLY_TX_FLAG_USE_WITNESS, serialized, len, &written);
```

Comment:
at this point you really could just initialize the flag to `WALLY_TX_FLAG_USE_WITNESS`, no?

Reply:
Yep.